packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/ExistingInvoiceDetails.tsx

```diff
@@ -373,3 +344,47 @@ const ExistingInvoiceDetailsBase = (props: ExistingReceivableDetailsProps) => {
     </>
   );
 };
+
+const InvoiceFileViewer = memo(({ invoiceId }: { invoiceId: string }) => {
+  const { data, isLoading, error, refetch } = usePDFReceivableById(invoiceId);
+  const { i18n } = useLingui();
+
+  if (error)
```

Comment:
`FileViewer` already handles error if `url` is not provided. So I think we don't need to handle an alert for it because `FileViewer` will show an error and an ability to `Refresh` request if user clicks on the button

Reply:
I'm not sure if `FileViewer` displays the error that occurs during the query. The query may crash, for example, why not tell the user about it?

Reply:
If `url` is not provided FileViewer will show an error. Or if PDF is not provided, or if PDF is not valid, etc
It has been made here: https://github.com/team-monite/monite-sdk/pull/71